Software/app/secure_element_lorawan/sys_app.c

```diff
@@ -72,11 +72,8 @@ void SystemApp_Init(void)
   DBG_Init();
 
   /*Initialize the terminal */

```

Comment:
Line is kept here (on others its removed), but maybe add a small space here?

Reply:
Done